services/src/main/java/org/exoplatform/officeonline/Constants.java

```diff
@@ -0,0 +1,235 @@
+package org.exoplatform.officeonline;
+
+/**
+ * The Class Constants.
+ */
+public class Constants {
+
+  /**
+   * Instantiates a new constants.
+   */
+  private Constants() {
+    // constants class
+  }
+
+  /** The Constant ACCESS_TOKEN_ATTRIBUTE. */
+  public static final String ACCESS_TOKEN_ATTRIBUTE              = "accessToken";
```

Comment:
I propose move WOPI only and internally used constants to wopi plugin package. Others general to the add-on could live on logic entities or service level.